whist/core/userlist.py

```diff
@@ -0,0 +1,66 @@
+"""
+Handles users joining and leaving a table.
+"""
+from whist.core.player import Player
+
+
+class UserList:
+    """
+    User handler for tables.
+    """
+    _users: dict[Player] = {}
+
+    def __len__(self):
+        return len(self._users)
+
+    @property
+    def ready(self) -> bool:
+        """
+        Returns if all players are ready.
+        :return: Ready or not
+        :rtype: boolean
+        """
+        for player_status in self._users.values():
+            if not player_status:
+                return False
+        return True
```

Comment:
can you fix #42 here?
maybe return the number of ready players instead?

Reply:
We can't fix #42 here because it has no knowledge about the min or max player. This is an issue with the table class.
Furthermore, I think the bug was related to something else. I think it because pydantic models are not reset between tests.

Reply:
> can you fix #42 here?
I added a test for this. It seems this PR is not responsible for it.